### PR TITLE
Milestone 5: Styling (Final Stage)

### PR DESCRIPTION
**Pull request by SaidRasinlic**
### In this milestone, I've completed the following things:

- Added  **look and feel** to React Components.
- Added **Progress** React Component to hold percentage status.
- Edited **README** with new project image.

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  min-height: 100vh;
+}
+
+.page-body {
+  padding: 1.438rem 6.188rem 1.688rem 6.25rem;
+}
+
+.menu-links {
+  list-style: none;
+  display: flex;
```
Comment (uche-inyama) [inline src/index.css:38]: These lines of code that is repeated in lines 37 & 38; 48 & 49 can be separated, defined and then passed as a class to the elements in the react components.

File: src/index.css [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  font-size: 25px;
+}
+
+.menu-links a {
+  text-decoration: none;
+}
+
+.books-list {
+  list-style: none;
+  display: flex;
```
Comment (uche-inyama) [inline src/index.css:49]: :point_up: Kindly refer to the earlier comment.

**Review by uche-inyama — CHANGES_REQUESTED** [review]
## Hi SaidRasinlic :slightly_smiling_face: 

Good job so far! Your project is coming along just fine. However, always endeavor to keep your css code (DRY)



## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by zainsadaqat — APPROVED** [review]
# APPROVED!!!! 🎉 🎉 🎉 🎉 🎉 🎉
## Hey @SaidRasinlic   😊 😃 

#### Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 

# Congratulations!!! 🎉 🥳 👏 🙌 🎊 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**